### ISSUE TITLE
Suggestion:why not in Cesium

### ISSUE THREAD

**Issue by kaktus40** [icholy/cesium_typescript_generator#1]
Hello,
I like using typescript and Cesium and your tool. Now that Cesium project use gulp for integration. Maybe that could be easy to incorporate your work?
For information, I think there would have more followers (AngularJS 2 use typescript!) :
https://groups.google.com/d/msg/cesium-dev/jGgNInY2Fqo/B8l6MmglFAAJ

Thanks for your generator!

**Comment by icholy** [comment]
There are some problems with how Cesium documents their inheritance. It doesn't give me enough information to generate complete definitions. I've brought up the issue to them and proposed a PR. That was about a year ago and they still haven't moved on it. I've given up and moved on from Cesium for the time being. I might revisit this project in the future though.

**Comment by icholy** [comment]
See https://github.com/jumpinjackie/jsdoc-typescript-plugin

**Comment by kaktus40** [comment]
Hello thanks for the reference!

**Comment by kaktus40** [comment]
Hello this is the latest generated 
[cesium.d.ts](https://github.com/icholy/cesium_typescript_generator/files/264924/cesium.d.txt) with you tool.
It remains some errors (lines 703, 1388 and 4248) where '*' should be 'any'.

**Comment by kaktus40** [comment]
Sorry, there is also some errors with the enum from Cesium like:
Intersect, ClockStep, ClockRange, ComponentDatatype, CornerType, PrimitiveType, ComponentDatatype, TimeStandard, ReferenceFrame, InterpolationAlgorithm, ExtrapolationType, Packable, HeightReference, HorizontalOrigin, ...

**Comment by icholy** [comment]
Yes, I know. The tool generates a starting point which needs to be tweaked manually. I never finished it because the Cesium documentation isn't good enough.
